django_auth_lti/backends.py

```diff
@@ -38,11 +35,11 @@ def authenticate(self, request):
             logger.error("Request doesn't contain an oauth_consumer_key; can't continue.")
             return None
 
-        if not oauth_creds:
+        if not settings.LTI_OAUTH_CREDENTIALS:
             logger.error("Missing LTI_OAUTH_CREDENTIALS in settings")
             raise PermissionDenied
 
-        secret = oauth_creds.get(request_key, None)
+        secret = settings.LTI_OAUTH_CREDENTIALS.get(request_key)
```

Comment:
The default return value for `.get` is None, so I thought to be more concise here.  I'd be fine adding it back in if people want, however.